### PR TITLE
Add core animation quiz to the readme.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -277,6 +277,147 @@ MotionAnimator is a purely Core Animation-based animator. If you are looking for
...
+RunLoop.main.run(mode: .defaultRunLoopMode, before: .distantFuture)
+
+UIView.animate(withDuration: 0.8, animations: {
+  layer.opacity = 0.5
+})
+```
+
+<details>
+  <summary>Click to see the answer</summary>
+  Generates an animation with duration of 0.25.
```
Comment (samnm) [inline README.md]: Is 0.25 correct here?

File: README.md [hunk 2]
```diff
@@ -277,6 +277,147 @@ MotionAnimator is a purely Core Animation-based animator. If you are looking for
...
+> - [Building Interruptible and Responsive Interactions](http://asciiwwdc.com/2014/sessions/236)
+> - [Advanced Graphics and Animations for iOS Apps](http://asciiwwdc.com/2014/sessions/419)
+> - [Advances in UIKit Animations and Transitions](http://asciiwwdc.com/2016/sessions/216)
+> - [Animating Layer Content](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CoreAnimation_guide/CreatingBasicAnimations/CreatingBasicAnimations.html)
+> - [Advanced Animation Tricks](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CoreAnimation_guide/AdvancedAnimationTricks/AdvancedAnimationTricks.html)
+> - [Additive animations: animateWithDuration in iOS 8](http://iosoteric.com/additive-animations-animatewithduration-in-ios-8/)
+
+There are two primary ways to animate with Core Animation on iOS:
+
+1. **implicitly**, with the UIView `animateWithDuration:` APIs, or by setting properties on unhosted CALayer instances, and
```
Comment (samnm) [inline README.md]: Unhosted CALayers? If I'm not mistaken the examples show setting properties on _hosted_ CALayer instances implicitly animates.

**Review by samnm — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (jverkoey) [reply README.md]: Yep!

```swift

func testDoesImplicitlyAnimateInUIViewAnimateBlock() {
  UIView.animate(withDuration: 0.5) {
    self.layer.opacity = 0.5
  }

  let animation = layer.animation(forKey: "opacity") as! CABasicAnimation
  XCTAssertEqual(animation.keyPath, "opacity")
  XCTAssertEqual(animation.duration, CATransaction.animationDuration()) // 0.25
}
```

Super fun, right?

File: README.md (hunk 2 above)
Comment (jverkoey) [reply README.md]: Clarified this.

File: README.md [hunk 3]
```diff
@@ -277,6 +277,147 @@ MotionAnimator is a purely Core Animation-based animator. If you are looking for
...
+> - [Building Interruptible and Responsive Interactions](http://asciiwwdc.com/2014/sessions/236)
+> - [Advanced Graphics and Animations for iOS Apps](http://asciiwwdc.com/2014/sessions/419)
+> - [Advances in UIKit Animations and Transitions](http://asciiwwdc.com/2016/sessions/216)
+> - [Animating Layer Content](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CoreAnimation_guide/CreatingBasicAnimations/CreatingBasicAnimations.html)
+> - [Advanced Animation Tricks](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CoreAnimation_guide/AdvancedAnimationTricks/AdvancedAnimationTricks.html)
+> - [Additive animations: animateWithDuration in iOS 8](http://iosoteric.com/additive-animations-animatewithduration-in-ios-8/)
+
+There are two primary ways to animate with Core Animation on iOS:
+
+1. **implicitly**, with the UIView `animateWithDuration:` APIs, or by setting properties on standalone CALayer instances (those that are **not** backing a UIView), and
```
Comment (samnm) [inline README.md:296]: Is this not still wrong?

This example implicitly animates a property.
```swift
let view = UIView()
window.addSubview(view)
let layer = CALayer()
view.layer.addSublayer(layer)

// Pump the run loop once.
RunLoop.main.run(mode: .defaultRunLoopMode, before: .distantFuture)

layer.opacity = 0.5
```

Comment (samnm) [reply README.md:296]: Oh I guess it isn't backing the view. Ignore me.

File: README.md (hunk 1 above)
Comment (samnm) [reply README.md]: Maybe add a note that makes it explicit that the docs are correct? I suspect most people would just think we have an error in the README here.

File: README.md (hunk 3 above)
Comment (samnm) [reply README.md:296]: To make this comment thread useful does the following implicitly animate?

```swift
let view = UIView()
window.addSubview(view)

// Pump the run loop once.
RunLoop.main.run(mode: .defaultRunLoopMode, before: .distantFuture)

view.layer.opacity = 0.5
```

Comment (jverkoey) [reply README.md:296]: It would not.

File: README.md (hunk 1 above)
Comment (jverkoey) [reply README.md]: Ah, good call. Will clarify.

**Comment by jverkoey** [comment]
Needs re-review.

**Review by romoore — APPROVED** [review]

**Pull request merged**